lib/pages/chat_list/chat_list_view_builder.dart

```diff
@@ -18,11 +20,14 @@ class ChatListViewBuilder extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return ListView.builder(
+    return ListView.separated(
       shrinkWrap: true,
       physics: const NeverScrollableScrollPhysics(),
-      itemCount: rooms.length,
+      itemCount: rooms.length + 1,
```

Comment:
Why need it?

Reply:
To have separator under the last element